src/ankh/cmd/ankh/main.go

```diff
@@ -261,3 +272,43 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
+		log.Infof("Running script: %s", path)
+		if ctx.DryRun == true {
+			log.Infof("- OK (dry) %s", path)
+			break
+		}
+		// pass kube context and the "global" config as a yaml environment variable
+		// also send the kube context as the first argument to the script
+		cmd := exec.Command(path, ctx.AnkhConfig.CurrentContext.KubeContext)
+		if ctx.AnkhConfig.CurrentContext.Global != nil {
+			global, err := yaml.Marshal(ctx.AnkhConfig.CurrentContext.Global)
+			check(err)
+			cmd.Env = append(
+				os.Environ(),
+				"ANKH_CONFIG_CONTEXT=" + string(global),
+				"ANKH_KUBE_CONTEXT=" + string(ctx.AnkhConfig.CurrentContext.KubeContext))
+		}
+		var stdOut, stdErr bytes.Buffer
+		cmd.Stdout = &stdOut
+		cmd.Stderr = &stdErr
+		err := cmd.Run()
+		if err != nil {
+			log.Fatalf("- FAILED %s:\nstdout: %s\nstderr: %s", path, stdOut.String(), stdErr.String())
+			log.Fatalf("Cannot proceed: %v\n", err)
```

Comment:
it's ok to omit this second Fatalf

Reply:
Cool wasn't sure if I needed to keep exact parity with the old logs, will remove